administration/src/bp-modules/project-settings/ProjectSettingsController.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactElement, useContext } from 'react'
+import styled from 'styled-components'
+
+import { WhoAmIContext } from '../../WhoAmIProvider'
+import { Role } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import UserEndpointSettings from './UserEndpointSettings'
+
+const ProjectSettingsContainer = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+`
+
```

Comment:
Here is a check for the site access missing.
```js
  if (role !== Role.ProjectAdmin) {
    return (
      <NonIdealState
        icon='cross'
        title='Fehlende Berechtigung'
        description={
          'Sie sind für die Projekteinstellungen nicht berechtigt'
        }
      />
    )
  }
```

Reply:
this is also not resolved. A different role can still access `/project` route (even empty)